packages/client-core/i18n/en/editor.json

```diff
@@ -242,6 +242,7 @@
       "lbl-textureType": "Texture Type",
       "lbl-textureUrl": "Texture URL",
       "lbl-intensity": "Envmap Intensity",
+      "info-intensity": "Envmap intensity may vary depending on material, shader, and environment map type configuration.",
```

Comment:
Maybe replace `Envmap` with `Visual` sounds a little clearer to me?

`Visual intensity may vary depending on material, shader, and environment map type configuration.`